src/js/components/Drop/DropContainer.js

```diff
@@ -176,15 +176,14 @@ export class DropContainer extends Component {
         maxHeight = windowHeight - top;
       } else if (align.bottom) {
         if (align.bottom === 'bottom') {
-          // top = targetRect.bottom - containerRect.height;
-          // maxHeight = Math.min(targetRect.bottom - top, windowHeight - top);
+          top = targetRect.bottom - containerRect.height;
```

Comment:
Actually, if the drop is anchored at the bottom, then `maxHeight = bottom` sounds right. We want the max-height to allow the drop to grow as much as will fit within the visible window area. Does that make sense?

Reply:
let me try this again. in my case i have `align.bottom === 'top'` so that the bottom of the drop is at the top of the target. this is the `else` scenario